docs/products/metrics/howto/download-influxdb-data.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Download Aiven for InfluxDB data backup
+sidebar_label: Download InfluxDB data
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+import ConsoleIcon from "@site/src/components/ConsoleIcons"
+
+Download a backup of your Aiven for InfluxDB service data by requesting a backup. This creates a snapshot of your latest data, ready for download.
+
+## Prerequisites
+
+Before you download a backup, make sure you have the following:
+
+- [Aiven API token](/docs/platform/howto/create_authentication_token) with the necessary
+  permissions
+- [Aiven CLI](/docs/tools/cli)
+
+If your InfluxDB service is in a Virtual Private Cloud (VPC) or if you need specific
+network access to your backup, confirm that the correct access settings are enabled.
+Based on your network setup, enable one of the following settings:
+
+- `public_access.user_backup`: Allows access to backups from the public internet
+- `private_access.user_backup`: Allows access to backups within a private network
+- `privatelink_access.user_backup`: Allows access to backups via a private link
+
+To configure access:
+
+1. Go to the service's <ConsoleLabel name="overview" /> page.
+1. Click <ConsoleLabel name="service settings" />, then go to **Cloud and network**.
+1. Click the <ConsoleLabel name="actions" /> menu and select
+   **More network configurations**.
+1. Click <ConsoleIcon name="Add config options"/>.
+1. In the search field, enter the access setting to enable. For example,
+   `private_access.user_backup`.
+1. Select the value to **Enabled**.
+1. Click **Save configuration**.
+
+## Backup format
+
+The backup created is a portable file that includes all databases and retention policies
```

Comment:
That is just a small detail and probably is not important, but InfluDB docs called it "portable format".